resources/artifact-ignores.properties

```diff
@@ -709,3 +709,6 @@ yammer = https://www.jenkins.io/blog/2021/12/22/deprecated-ruby-runtime/
 # https://www.jenkins.io/security/advisory/2022-02-15/
 team-views = https://www.jenkins.io/security/plugins/#suspensions
 sinatra-chef-builder = https://www.jenkins.io/security/plugins/#suspensions
+
+# The plugin has been closed source since 2015
+starteam = https://github.com/jenkins-infra/update-center2/pull/571
```

Comment:
```suggestion
starteam = https://wiki.jenkins.io/JENKINS/StarTeam
```

Reply:
Without the "display" in the wiki URL there is no redirection to plugins.jenkins.io

Reply:
> Without the "display" in the wiki URL there is no redirection to plugins.jenkins.io

I think this pull request is the better justification for suspending distribution, rather than pointing to the wiki page.  I find this pull request more easily read than the wiki page.

Reply:
I've suggested this as pointing to this plugin wiki page without being redirected to plugins.jenkins.io was a request from @daniel-beck on IRC.